lib/agenda/find-and-lock-next-job.ts

```diff
@@ -54,21 +54,17 @@ export const findAndLockNextJob = async function (
     // * @type {{$and: [*]}}
     // */
     const JOB_PROCESS_WHERE_QUERY = {
-      $and: [
+      $or: [
         {
           name: jobName,
           disabled: { $ne: true },
+          lockedAt: { $eq: null },
```

Comment:
So that's the condition that is supposed to catch both a case of null lockedAt and none existing lockedAt ?